README.md

```diff
@@ -1,10 +1,13 @@
 # Minesweeper
 
-is a single player puzzle game. The object of the game is to clear a rectangular board containing hidden "mines" or bombs without detonating any of them, using the guidance on the number of adjacent mines in each field. The game dates from the 1960s
+A single-player puzzle game. You need to clear a rectangular board containing hidden "mines" or bombs without detonating any of them, using the guidance on the number of adjacent mines in each field. The game dates from the 1960s
 
-In Minesweeper, mines are scattered throughout a board which is divided into cells. Cells have three states: uncovered, covered and flagged. A covered cell is blank and clickable, while an uncovered cell is exposed. Flagged cells are those marked by the player to indicate a potential mine location.
+Mines are scattered throughout a board, which is divided into cells. Cells have three states:
```

Comment:
Good lists can transform technical chaos into something orderly. Technical readers generally love lists. Therefore, when writing, seek opportunities to convert prose into lists.